docker/julia/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
If it's not strictly necessary, try to avoid using a custom entrypoint. Instead, you can use the post-init.sh script like we do for setting up the R image: https://github.com/SwissDataScienceCenter/renkulab-docker/blob/master/docker/r/post-init.sh - this is executed by the entrypoint from the base image [here](https://github.com/SwissDataScienceCenter/renkulab-docker/blob/master/docker/py/entrypoint.sh#L40)